Windows/main.cpp

```diff
@@ -280,6 +280,19 @@ static int ScreenDPI() {
 #endif
 #endif
 
+static int ScreenRefreshRateHz() {
+	DEVMODE lpDevMode;
+	memset(&lpDevMode, 0, sizeof(DEVMODE));
+	lpDevMode.dmSize = sizeof(DEVMODE);
+	lpDevMode.dmDriverExtra = 0;
+
+	if (EnumDisplaySettings(NULL, ENUM_CURRENT_SETTINGS, &lpDevMode) == 0) {
+		return 60;  // default value
+	} else {
+		return lpDevMode.dmDisplayFrequency > 15 ? lpDevMode.dmDisplayFrequency : 60;
```

Comment:
Hm, should we check `DM_DISPLAYFREQUENCY` on `dmFields`?  I guess it's probably not needed.

Also, personally I prefer not putting a dangling return inside an else in a case like this.

-[Unknown]